src/Core/Application/Exception/InvalidTaskTypeException.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Exception;
+
+use Exception;
+
+class InvalidTaskTypeException extends Exception
```

Comment:
Why this exception in application layer when the rest logic in infrastructure.

Reply:
because this exception can be caught in other places.